oaknational/Oak-Web-Application
# [CUR-902] Maths KS4 tiers, removed duplicates and improved unit specificity

## Description

Music year: 1966

- Added additional filtering to the getStaticProps curriculum units data.

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2846--oak-web-application.netlify.thenational.academy/teachers/curriculum/maths-secondary/units
2. Goto year 10 foundation
3. You should not see Circle theorems
4. Click higher filter
5. You should see Circle theorems

--

1. Go to https://deploy-preview-2846--oak-web-application.netlify.thenational.academy/teachers/curriculum/science-secondary-edexcel/units
2. Combined science and other units should be the same as CAT and the live site

## Checklist
